applications/GeoMechanicsApplication/custom_elements/transient_Pw_element.cpp

```diff
@@ -430,12 +418,11 @@ void TransientPwElement<TDim,TNumNodes>::
         if ( rOutput.size() != mRetentionLawVector.size() )
             rOutput.resize(mRetentionLawVector.size());
 
-        for ( unsigned int i = 0;  i < mRetentionLawVector.size(); i++ )
-        {
-            rOutput[i] = mRetentionLawVector[i]->GetValue( rVariable, rOutput[i] );
+        for ( unsigned int i = 0;  i < mRetentionLawVector.size(); ++i ) {
```

Comment:
you could use `std::fill` here (https://en.cppreference.com/w/cpp/algorithm/fill)